### PR TITLE
Update Feedforward to new names and values

### PR DESCRIPTION
**Pull request by Asizon**
This PR goes with small part of https://github.com/betaflight/betaflight/pull/10727
Updates names and some values for the recent added feedforward settings, since this is added in 1.44 no msp changes needed.
Wainting for some names/style reviewing :)

image

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> FAIL
 - PR is not in draft -> PASS
 - Assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - 'Don't merge' label NOT found -> PASS
 - At least one 'RN:' label found -> FAIL
 - 'Tested' label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

File: src/js/tabs/pid_tuning.js
Comment (haslinghuis) [inline src/js/tabs/pid_tuning.js:485]: Maybe change all `ff` to `feedforward`

**Review by haslinghuis — CHANGES_REQUESTED** [review]

**Review by haslinghuis — DISMISSED** [review]
LGTM!

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> FAIL
 - PR is not in draft -> PASS
 - Assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - 'Don't merge' label NOT found -> PASS
 - At least one 'RN:' label found -> PASS
 - 'Tested' label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

File: locales/en/messages.json
Comment (klutvott123) [inline locales/en/messages.json:1801]: This should be "Off"

**Comment by klutvott123** [comment]
I think it would make sense now to group these new feedforward settings together with feedforward transition.
Like this https://user-images.githubusercontent.com/41271048/119407111-0f49eb00-bce4-11eb-9398-af7670a2a6f2.png

**Comment by Asizon** [comment]
Yes @klutvott123 you are right in your all review, lets modify it!

**Comment by Asizon** [comment]
Here is the new result
image

**Comment by Asizon** [comment]
Ouch i have removed another diferent feedforward option from 118n, I think we have now more than one feedforward parameters, I need to rename one.

**Comment by Asizon** [comment]
image

**Comment by McGiverGim** [comment]
Sorry for getting to this too late... This breaks totally the style of the page. I think this goes better in the left pane, where we have a merge of different things... And we had the old ff transition.

**Comment by McGiverGim** [comment]
Another solution is to add a slider to disable ff that hides it in the pid and gives it a zero value...

**Comment by Asizon** [comment]
yes @McGiverGim , i thought about adding it under motor output limit. I'll do it and see what it looks like. If anyone has any new design ideas for this.
For the slider, i think now we cant disable feedforward in this way, @ctzsnooze what do you think?

**Comment by McGiverGim** [comment]
All the other elements have an slider (check) to enable or disable the feature. I know it's a little silly to disable ff, but it will be coherent with the other elements.

**Comment by Asizon** [comment]
Yes but Feedforward is permanetly enabled now by USE_FEEDFORWARD on firmware.

**Comment by McGiverGim** [comment]
I suppose that if we give it a zero value, it will be deactivated. But I don't know if it is possible with all this sliders modifying the value for it's own... I suppose that not. I don't know what is the best solution for this...

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> FAIL
 - PR is not in draft -> PASS
 - Assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - 'Don't merge' label NOT found -> PASS
 - At least one 'RN:' label found -> PASS
 - 'Tested' label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> FAIL
 - Assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - 'Don't merge' label NOT found -> PASS
 - At least one 'RN:' label found -> PASS
 - 'Tested' label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by McGiverGim** [comment]
Another option... maybe is a silly way... add the slider/check but let it locked in the enabled position. Add information to the tooltip about that it is enabled in the firmware part and can't be disabled (or a modal warning when the user tries to disable it).
In this way it will look like the others elements.

**Comment by Asizon** [comment]
@McGiverGim I have discussed just now with Chris, and maybe we can back on/off switch again, that disables feedforward group, goes pid F to 0 and hide F slider.

**Comment by McGiverGim** [comment]
I let this on your hands, only were suggestions trying to find a good way to add this. If it can be disabled, better for the Configurator.

**Comment by Asizon** [comment]
@McGiverGim i have made some progress on this, if you want to see the new way to add the switch. Not yet finished.

**Comment by Asizon** [comment]
Ok so this PR is ready for review.
Now user can disable Feedforward with a switch and all pid F values go 0 to disable all feedforward feature. When user reenable feedforward it puts again version Api based F values.
Note that this way only works with our current sliders disabled, because the current implementation doesnt support disable F to 0 and continue using sliders.

**Comment by Asizon** [comment]
Betaflight Configurator 2021-06-20 11-15-15

**Review by haslinghuis — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

File: src/js/tabs/pid_tuning.js
Comment (haslinghuis) [inline src/js/tabs/pid_tuning.js:37]: Need to check why it was not needed before

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> FAIL
 - Assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> FAIL
 - 'Don't merge' label NOT found -> PASS
 - At least one 'RN:' label found -> PASS
 - 'Tested' label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

File: src/js/tabs/pid_tuning.js
Comment (Asizon) [reply src/js/tabs/pid_tuning.js:37]: Because we never haven't used fc.js so far to interact with PID defaults here, time to do this with all defaults in a future PR.

**Review by McGiverGim — APPROVED** [review]
Seems ok to me. Thanks for the changes. It's not perfect, but when the things in the firmware are not done thinking in the Configurator, and we need to maintain old things too, there is not too much that can be done.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> FAIL
 - Assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - 'Don't merge' label NOT found -> PASS
 - At least one 'RN:' label found -> PASS
 - 'Tested' label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> FAIL
 - Assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - 'Don't merge' label NOT found -> PASS
 - At least one 'RN:' label found -> PASS
 - 'Tested' label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by limonspb — APPROVED** [review]

**Review by klutvott123 — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> PASS
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (PASS)
 - github identifies PR as mergeable -> PASS
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> PASS

**Pull request merged**
